### PR TITLE
Cookies examples update

### PR DESCRIPTION
**Pull request by cjrace**
I've added a single example of how to use the cookies functions in a minimal shiny app that we can reuse across the documentation for all of our cookies family of functions. Main change here is in the cookies.R script - expand the diff there to review

While doing so I also took the liberty of adding in more default arguments so that analysts don't need to add as much code, so I've made the defaults what was suggested and then removed those arguments from the examples to encourage analysts to just keep the defaults.

- I tried to do this with the `reactive(input$cookies_input)` arguments too but they wouldn't work as defaults only and seemed to need to be explicit, so I've left those in the server file

Additional notes:

- #41 adds a GitHub action that is now required for all PRs to main, this isn't on this branch so I'll just force a merge over that if this PR is approved before that one (if it's not then I'll be updating this branch to have it in once that gets to main)
- Lots of `lintr` notes, expecting them to be resolved in #41 separately so ignoring them intentionally here

### CODE REVIEW

**Comment by cjrace** [comment]
[Update cookies script examples in dfeshiny](https://trello.com/c/mFy2p0Ff/1569-update-cookies-script-examples-in-dfeshiny)

**Comment by jen-machin** [comment]
image

Some of the examples end up inside a box with a scrollbar on the documentation site - is there any way around this?

File: R/cookies.R [hunk 1]
```diff
@@ -1,10 +1,55 @@
 #' dfe_cookie_script
 #'
+#' Calls in JavaScript dependencies to the shiny app. Function should be placed
```
Comment (jen-machin) [inline R/cookies.R:3]: Can we add a sentence about what the JavaScript dependencies do?

File: R/cookies.R [hunk 2]
```diff
@@ -1,10 +1,55 @@
...
+#'   )
+#'
+#'   # Server logic for the panel, can be placed anywhere in server.R ---------
+#'   cookies_panel_server(
+#'     input_cookies = reactive(input$cookies),
+#'     google_analytics_key = ga_key
+#'   )
+#' }
+#'
+#' # How to run the minimal app give in this example ==========================
```
Comment (jen-machin) [inline R/cookies.R:48]: is this meant to be "given" rather than "give"?

File: R/cookies.R [hunk 3]
```diff
@@ -243,196 +279,176 @@ init_cookies <- function() {
...
-            width = 12,
-            shiny::tags$h1("Cookies"),
-            shiny::tags$p("Cookies are small files saved on your phone, tablet or
+#' cookies_panel_ui
+#'
+#' @description
+#' Create the standard DfE R-Shiny cookies dashboard panel in the ui. The server
+#' side functionality is provided by cookies_panel_server()
+#'
+#' @param id ID shared with cookies_panel_server()
```
Comment (jen-machin) [inline R/cookies.R:288]: Can we add a bit more explanation about the ID? Does the user decide what it's called? How do they decide what to call it?

**Comment by cjrace** [comment]
> image
> Some of the examples end up inside a box with a scrollbar on the documentation site - is there any way around this?

Ah yeah, that was my bad, I forgot to wrap the example in a way that stopped it actually running, should be better now?

File: R/cookies.R (hunk 3 above)
Comment (cjrace) [reply R/cookies.R]: Added, and done this consistently across all four on the page that use an id argument

File: R/cookies.R (hunk 2 above)
Comment (cjrace) [reply R/cookies.R]: Yep! Good spot, fixed now

File: R/cookies.R (hunk 1 above)
Comment (cjrace) [reply R/cookies.R]: Have attempted one!

**Review by jen-machin — APPROVED** [review]
All looks good!

**Pull request merged**